runtime/jcl/cl_se7_basic/CMakeLists.txt

```diff
@@ -22,21 +22,20 @@
 
 
 set_source_files_properties(${CMAKE_CURRENT_BINARY_DIR}/../ut_j9jcl.c PROPERTIES GENERATED TRUE)
-add_library(jclse7b_ SHARED
-	../j9vmconstantpool_se7_basic.c
-)
-target_include_directories(jclse7b_ 
+
+add_jcl(se7_basic)
+target_include_directories(jclse7_basic_
 	PRIVATE
 		${CMAKE_CURRENT_BINARY_DIR}
 		../
 )
 
-target_compile_definitions(jclse7b_
+target_compile_definitions(jclse7_basic_
 	PRIVATE
 		J9VM_JCL_SE7_BASIC
 )
 
-target_link_libraries(jclse7b_
+target_link_libraries(jclse7_basic_
```

Comment:
If this implies changing the library name from `libjclse7b_29.so` to `libjclse7_basic_29.so` then it will break Java 8 builds.

Reply:
It would, but a couple lines down I have

`set_target_properties(jclse7_basic_ PROPERTIES LIBRARY_OUTPUT_NAME jclse7b_)`

Reply:
Is there some benefit to the extra name (`jclse7_basic_`)? If, not, please just use `jclse7b_ ` consistently.

Reply:
basically to simplify the logic of `add_jcl`  so that it can assume the constant pool file is named `j9vmconstantpool_<NAME>.c`

Reply:
That makes sense. Please add a comment to that effect.

I hope that in the not-to-distant future, we'll only have to build one jclse library.

Reply:
But for now, this should include`CMakeLists.txt` for `jcl/cl_se11` and `jcl/cl_se12`.